Data-Collection-Inventory/2021Q2_csvs/data_collection_reported_specimen_count.csv

```diff
@@ -0,0 +1,102 @@
+feng_fish,cell in slice,145
+feng_fish,cell in slice,145
+feng_morph,whole cell,45
+feng_sn_10xv3,cell nucleus,344746
+feng_sn_dropseq,cell nucleus,834367
+fischl_lsfm,tissue sample,29
+fischl_mri,brain hemisphere,9
+fischl_oct,brain slice,128
+fischl_stereology,tissue sample,16
+huang_antero,brain section set,49
+huang_fmost,whole brain,164
+huang_fmost_cells,whole cell,467
+huang_input_conn,whole brain,5
+huang_pn_bulk_SSv4,bulk nucleus,19
+huang_pn_sn_10xv2,cell nucleus,106093
+huang_pn_sn_10xv3,cell nucleus,157696
+huang_retro,brain section set,50
+huang_sn_10xv2,cell nucleus,469176
+huang_sn_10xv3,cell nucleus,5477265
+huang_stpt_conn,whole brain,232
+huang_stpt_count,whole brain,58
+human_cortex_SS4_Open_GRU,cell nucleus,4911
+human_cortex_SS4_restricted_GRU,cell nucleus,50688
+human_cortex_SS4_restricted_limited,cell nucleus,208
+kriegstein_sc_10xv2,whole cell,736509
+kriegstein_sc_10xv3,whole cell,259015
+kriegstein_sc_ATACseq,cell nucleus,156334
+kriegstein_sn_10xv2,cell nucleus,1823807
+lein_sn_10xv2,cell nucleus,180766
+lein_sn_10xv3,cell nucleus,1709430
+lein_sn_SSv4,cell nucleus,244983
+nowakowski_evobc,whole cell,20424
+nowakowski_pndev_10xv3,cell nucleus,124580
+osten_stpt,whole brain,393
+regev_cisi,brain slice,1
+tasic_ATACseq,whole cell,13093
+tasic_stpt,whole brain,40
+tilgner_sc_p56,cell nucleus,38311
+yang_morf_cells,whole cell,1957
+yang_MORF_confocal,brain section set,22
+zeng_anderson_sc_10xv2,whole cell,168877
+zeng_expr_char,whole brain,53
+zeng_fmost,whole brain,141
+zeng_fmost_cells,whole cell,1698
+zeng_mouselight,whole brain,1
+zeng_retro,whole brain,746
+zeng_sc_10xv2,whole cell,2459753
+zeng_sc_10xv3,whole cell,1315525
+zeng_sc_SSv4,whole cell,156374
+zeng_sn_10xv2,cell nucleus,90902
+zeng_sn_10xv3,cell nucleus,90382
+zeng_sn_SSv4,cell nucleus,6816
+zeng_tolias_pseq_e,cell in slice,1320
+zeng_tolias_pseq_m,cell in slice,1320
+zeng_tolias_pseq_t,cell nucleus,1320
+zhangK_cj_snareseq_AC,cell nucleus,10780
+zhangK_cj_snareseq_R,cell nucleus,12232
+zhangK_hs_snareseq_AC,cell nucleus,159166
+zhangK_hs_snareseq_R,cell nucleus,170224
+zhangl_anatomy,spinal cord slice,11
+zhangl_kz_omics,cell nucleus,23376
+zhangl_seqfish,spinal cord slice,4
+dulac_pag_sn_10xv2,cell nucleus,116659
+dong_confocal,brain slice,29
+dong_lightsheet,whole brain,2
+huang_dev_cx_scATAC,brain region,8016
+huang_dev_cx_scRNA,brain region,3
+lein_evo_10x_arcticgroundsquirrel,cell nucleus,11
+lein_evo_10x_armadillo,cell nucleus,12
+lein_evo_10x_chimp,cell nucleus,12
+lein_evo_10x_ferret,cell nucleus,12
+lein_evo_10x_gorilla,cell nucleus,14
+lein_evo_10x_greenmonkey,cell nucleus,12
+lein_evo_10x_nemestrina,cell nucleus,12
+lein_evo_10x_opposum,cell nucleus,12
+lein_evo_10x_owlmonkey,cell nucleus,12
+lein_evo_10x_rabbit,cell nucleus,13
+lein_evo_10x_rat,cell nucleus,12
+lein_evo_10x_rhesus,cell nucleus,13
+mop_evo_pseq_hu_t,cell nucleus,4
+zhuang_dulac_merfish_MOE,whole cell,3300000
+zhuang_dulac_merfish_MOE,whole cell,3300000
```

Comment:
several of these have duplicate entries

Reply:
This is a bug, which I think is related to `Reported count` being associated directly with a data collection object and not a specimen count object. I'm manually deleting these for now until we decide whether we want to fix the bug or restrict data collections to only one specimen type.